lib/routes/index.js

```diff
@@ -59,6 +60,7 @@ module.exports = function (
   )
   const session = require('./session')(log, isA, error, db)
   const sign = require('./sign')(log, P, isA, error, signer, db, config.domain, devices)
+  const smsRoute = require('./sms')(log, isA, error, config, customs, smsImpl)
```

Comment:
There are two things here which could reasonably lay claim to the name `sms`: the route and the implementation exported from the mailer.

The other routes don't have this problem because they don't receive a variable that logically has the same name as them. By suffixing both of them I was hoping to eradicate any chance of mixing up which is which.

Any suggestions for better names? Just `sms` and `smsImpl` seems ambiguous to me.